src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTask.java

```diff
@@ -74,22 +73,36 @@ private void processZipFile(CloudBlobContainer container, String zipFilename)
         log.info("Processing zip file {}", zipFilename);
 
         CloudBlockBlob cloudBlockBlob = container.getBlockBlobReference(zipFilename);
-        BlobInputStream blobInputStream = cloudBlockBlob.openInputStream();
-
-        //Zip file will include metadata.json and collection of pdf documents
-        try (ZipInputStream zis = new ZipInputStream(blobInputStream)) {
-            ZipFileProcessor zipFileProcessor = processZipInputStream(zis, zipFilename, container.getName());
-
-            if (zipFileProcessor != null) {
-                processParsedEnvelopeDocuments(
-                    zipFileProcessor.getEnvelope(),
-                    zipFileProcessor.getPdfs(),
-                    cloudBlockBlob
-                );
+
+        CloudBlockBlob blobWithLeaseAcquired = acquireLease(cloudBlockBlob, container.getName(), zipFilename);
+
+        if (Objects.nonNull(blobWithLeaseAcquired)) {
+            BlobInputStream blobInputStream = blobWithLeaseAcquired.openInputStream();
+
+            //Zip file will include metadata.json and collection of pdf documents
```

Comment:
another missing space

Reply:
comment was not part of my PR changes any ways will update it :)